twogplus/views.py

```diff
@@ -46,7 +45,7 @@ def upload_cert():
         else:
             message = str(e)
         flash(message, "danger")
-        return render_template("home.html")
+        return home()
```

Comment:
What is the reason that you replaced `render_template("home.html")` with home? Does this change the behaivor of the app at all?

Reply:
Yes it does. It also shows uploaded tests and vaccinations in case of an erroneous upload. (behaviour before changing that line was that you needed to reload the page afterwards)